src/sos-packaging.props

```diff
@@ -56,11 +50,25 @@
     <SosRequiredBinaries Include="$(ArtifactsBinDir)\osx.arm64.$(Configuration)\libsos.dylib" TargetRid="osx-arm64" />
     <SosRequiredBinaries Condition="'$(PackageWithCDac)' == 'true'" Include="$(ArtifactsBinDir)\osx.arm64.$(Configuration)\libmscordaccore_universal.dylib" TargetRid="osx-arm64" />
     <SosRequiredBinaries Include="$(ArtifactsBinDir)\osx.arm64.$(Configuration)\sosdocsunix.txt" TargetRid="osx-arm64" />
+
+    <_SosBinariesToPack Include="@(SosRequiredBinaries)" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <_winSosRequiredBinaries Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\sos.dll" TargetRid="win-x64" />
+    <_winSosRequiredBinaries Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\Microsoft.DiaSymReader.Native.amd64.dll" TargetRid="win-x64" />
+    <_winSosRequiredBinaries Condition="'$(PackageWithCDac)' == 'true'" Include="$(ArtifactsBinDir)\Windows_NT.x64.$(Configuration)\mscordaccore_universal.dll" TargetRid="win-x64" />
+
+    <!-- When we are running the official build and packaging, don't copy too much. We already publish sos.dll in another leg, so favor that. -->
+    <_SosBinariesToPack Condition="'$(BuildX64Package)' == 'true'" 
+                        Include="@(_winSosRequiredBinaries)" />
+
+    <SosRequiredBinaries Include="@(_winSosRequiredBinaries)" />
```

Comment:
Do you happen to know why `Windows_NT.x64` is always included in `SosRequiredBinaries`?
Thinking if theres a better name since `_winSosRequiredBinaries` only covers Win x64, and someone in the future might wonder why win x86 and win arm64 aren't considered `_winSosRequiredBinaries`. So wondering if theress a different way to describe why `Windows_NT.x64` always gets included in `SosRequiredBinaries`

Reply:
We package on a win-x64 host and that's why it's defaulted.